sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -208,10 +208,10 @@ def test_dataframe_tests(self):
             ],
             'pandas.core.frame.DataFrame.sort_index': ['*'],
             'pandas.core.frame.DataFrame.sort_values': ['*'],
+            'pandas.core.frame.DataFrame.melt': ['*']
```

Comment:
Could you modify this so it just skips the calls that don't work? Unfortunately that will be most of them, since the defiault is `ignore_index=True`, but based on https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.melt.html it looks like we can at least run `df.melt(id_vars=['A'], value_vars=['B', 'C'], ignore_index=False)`